src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+const listOfEmployees = document.querySelector('ul');
+let items = document.querySelectorAll('li');
+
+function sortList(list) {
+  listOfEmployees.innerHTML = '';
+
+  const sortedList = [...list].sort((a, b) => {
+    const salaryA = Number(a.dataset.salary.slice(1).split(',').join(''));
+    const salaryB = Number(b.dataset.salary.slice(1).split(',').join(''));
```

Comment:
Look, you do the same actions several times. Would be great to create a method for converting to a number